app/templates/main/map.html

```diff
@@ -0,0 +1,72 @@
+<html>
+<head>
+<script src="http://maps.googleapis.com/maps/api/js"></script>
+<script>
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+function initialize() {
+var initial_coords = new google.maps.LatLng(39.952, -75.195);
+var mapProp = {
+center: initial_coords,
+zoom:17,
+mapTypeId: google.maps.MapTypeId.ROADMAP
+};
+map=new google.maps.Map(document.getElementById("googleMap"),mapProp);
+}
+function reinitialize() {
+var initial_coords = new google.maps.LatLng(39.952, -75.195);
```

Comment:
+1 to getting rid of "magic numbers" for zoom and lat/long (see http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)